packages/core/components/Puck/styles.module.css

```diff
@@ -35,9 +35,33 @@
 .Puck--leftSideBarVisible {
   grid-template-columns:
     var(--puck-side-bar-width) var(--puck-frame-width)
+    0;
+}
+
+.Puck--rightSideBarVisible {
+  grid-template-columns:
+    0 var(--puck-frame-width)
     var(--puck-side-bar-width);
 }
 
+.Puck--leftSideBarVisible.Puck--rightSideBarVisible  {
+  grid-template-columns:
+  var(--puck-side-bar-width) var(--puck-frame-width)
+    var(--puck-side-bar-width);
+}
+
+@media (min-width: 458px) {
+  .Puck {
+    --puck-frame-width: minmax(266px, auto);
+  }
+}
+
+@media (min-width: 638px) {
+  .Puck {
+    --puck-side-bar-width: minmax(186px, 250px);
```

Comment:
I wonder if we should actually introduce this a viewport down, at 458px, so the inputs are a bit more usable. I think the inputs are more important than the preview when editing.

Reply:
Spun out to #247